src/main/java/io/pakland/mdas/githubstats/shell/components/PullRequestComponent.java

```diff
@@ -0,0 +1,41 @@
+package io.pakland.mdas.githubstats.shell.components;
```

Comment:
I would probably not add this in the PR as it is something aside, but no issue this time 🙌🏼

Reply:
Sure! Sorry.